src/OpenLoco/Widget.cpp

```diff
@@ -996,4 +996,19 @@ namespace OpenLoco::Ui
             Gfx::drawRect(*ctx, pos.x, pos.y + 26, 31, 1, Colours::getShade(w->getColour(WindowColour::secondary).c(), 7));
         }
     }
+
+    void Widget::leftAlignTabs(Window& window, uint8_t firstTabIndex, uint8_t lastTabIndex, uint16_t tabWidth)
+    {
+        int16_t xPos = window.widgets[firstTabIndex].left;
+        for (uint8_t i = firstTabIndex; i <= lastTabIndex; i++)
+        {
+            if (window.isDisabled(i) == false)
+            {
+                window.widgets[i].type = WidgetType::tab;
```

Comment:
I think as this is required we should do the inverse as well and set it to type none for when the tab is disabled.